EU4ToVic2Tests/EU4WorldTests/LeaderTests/EU4LeaderTests.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Leader/EU4Leader.h"
+#include "gtest/gtest.h"
+
+TEST(EU4World_EU4LeaderTests, primitivesDefaultToBlank)
```

Comment:
Update test name to be more clear that not all the defaults are the same (but that everything is default)